### PR TITLE
[CDAP-13381] replacing overview with control center in 404 page

### PR DESCRIPTION
**Pull request by itsanudeep**
JIRA: https://issues.cask.co/browse/CDAP-13381

Build: https://builds.cask.co/browse/CDAP-UDUT497-1

### CODE REVIEW

File: cdap-ui/app/cdap/components/404/index.js [hunk 1]
```diff
@@ -48,7 +48,9 @@ export default function Page404({ entityName, entityType, children }) {
           <div className="navigation-section">
             <div>
               {T.translate(`${I18N_PREFIX}.subtitleMessage2`)}
-              <Link to={`/ns/${namespace}/`}>{T.translate(`${I18N_PREFIX}.overviewLabel`)}</Link>
+              <Link to={`/ns/${namespace}/control`}>
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/404/index.js]: i wonder if we should tell user to go to home instead of control center?

Comment (itsanudeep) [reply cdap-ui/app/cdap/components/404/index.js]: @yukiej also had the same comment. @bdmogal What do you think we should do?

Comment (itsanudeep) [reply cdap-ui/app/cdap/components/404/index.js]: Talked to Bhooshan, redirecting users to home page here.

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Pull request merged**